cross/tcl/PLIST

```diff
@@ -0,0 +1,257 @@
+rsc:bin/sqlite3_analyzer
```

Comment:
Generic for this file: please check the PLIST wiki article, it may contain some helpful information for you.